index.js

```diff
@@ -45,6 +45,7 @@ const safeObject = require('./util/safe_object');
  * @property {false} [vimeo] Disable Vimeo embeds.
  * @property {false} [twitter] Disable Twitter embeds.
  * @property {false} [instagram] Disable Instagram embeds.
+ * @property {false} [compare] Disable Compare embeds.
```

Comment:
```suggestion
 * @property {false} [compare] Disable Image Compare embeds.
```
Just for consistency in wording

Reply:
Updated.